recipes/r-saccharis/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+r_base:
+  - 3.6.3
```

Comment:
Why this old version ?
I see the upstream Git repo is well maintained (by you!).
Doesn't it support 4.x ?

Reply:
I've been having trouble with the solver because some of the dependencies seem to have out of date version compatibility with r_base. At the time I got the conda build working at all, 3.6.3 was the latest version that satisfied all version requirements between the dependencies. I have since removed one of the dependencies since it wasn't available in conda-forge or bioconda channels( only the r channel) and was not strictly needed, and it looks like I can update the build to version 4.3.*, and I just sent in another commit updating the conda_build_config file with that change. Hopefully everything else looks good to add this package otherwise.